deploy/olm-catalog/ocs-operator/manifests/storagecluster.crd.yaml

```diff
@@ -1440,6 +1440,9 @@ spec:
                             type: string
                         type: object
                     type: object
+                  devicetype:
+                    description: DeviceType is the value of device type in this StorageDeviceSet
```

Comment:
DeviceType represents the value of underlying storage device used (restricted to 3 values; SSD, NVME, HDD)